src/sentry/models/outbox.py

```diff
@@ -174,24 +177,26 @@ def process_coalesced(self) -> Generator[OutboxBase | None, None, None]:
         if coalesced is not None:
             first_coalesced: OutboxBase = self.select_coalesced_messages().first() or coalesced
             _, deleted = self.select_coalesced_messages().filter(id__lte=coalesced.id).delete()
-            tags = {"category": OutboxCategory(self.category).name}
-            metrics.incr("outbox.processed", deleted, tags=tags)
-            metrics.timing(
+
+            set_measurement(
                 "outbox.processing_lag",
                 datetime.datetime.now().timestamp() - first_coalesced.scheduled_from.timestamp(),
-                tags=tags,
+                "second",
             )
 
     def process(self) -> bool:
-        with self.process_coalesced() as coalesced:
-            if coalesced is not None:
-                with metrics.timer(
-                    "outbox.send_signal.duration",
-                    tags={"category": OutboxCategory(coalesced.category).name},
-                ):
-                    coalesced.send_signal()
-                return True
-        return False
+        with sentry_sdk.start_transaction(op="outbox.process", name="process") as transaction:
```

Comment:
```suggestion
        with sentry_sdk.start_transaction(op="outbox.process", name="outbox.process") as transaction:
```

Having the name be `outbox.process` is a slightly better name. 🤷